pkg/resources/diff_suppressions.go

```diff
@@ -30,17 +30,17 @@ func IgnoreAfterCreation(_, _, _ string, d *schema.ResourceData) bool {
 	return d.Id() != ""
 }
 
-func IgnoreChangeToCurrentSnowflakeValueInShow(keyInShowOutput string) schema.SchemaDiffSuppressFunc {
+func IgnoreChangeToCurrentSnowflakeValueInOutput(attrName, keyInShowOutput string) schema.SchemaDiffSuppressFunc {
```

Comment:
I think this and the function below should be renamed. This one is used for plain values and the function below for "describe properties" (could be also used for parameters I guess), so the names should be something like Ignore...Plain[or Flat]ValueInOutput and the function below Ignore...ParameterValueInOutput or something similar (names can be changed).

Reply:
Changed the first one to suggested by you, but changed the second one to `IgnoreChangeToCurrentSnowflakeListValueInDescribe` because it's used when field is `DescribePropertyListSchema`.